autoPyTorch/pipeline/components/training/losses.py

```diff
@@ -11,16 +11,17 @@
 from autoPyTorch.constants import BINARY, CLASSIFICATION_TASKS, CONTINUOUS, MULTICLASS, REGRESSION_TASKS, \
     STRING_TO_OUTPUT_TYPES, STRING_TO_TASK_TYPES, TASK_TYPES_TO_STRING
 
+
 losses = dict(classification=dict(
     CrossEntropyLoss=dict(
-        module=CrossEntropyLoss, supported_output_type=MULTICLASS),
+        module=CrossEntropyLoss, supported_output_types=(MULTICLASS, BINARY)),
     BCEWithLogitsLoss=dict(
-        module=BCEWithLogitsLoss, supported_output_type=BINARY)),
+        module=BCEWithLogitsLoss, supported_output_types=(BINARY,))),
```

Comment:
Hey, can you add a check in so that we make sure this is a list? Or can you instead of a tuple actually make a list? It is very easy to miss the comma...  Like this can be a purest making sure everything is a list or even an in code check

Reply:
yeah I'll make them lists, and add a check for that

Reply:
But as mentioned, if we input something like supported_output_type=[MULTICLASS], the argument is gonna be updated by this.